packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/EsxiCredentialsEdit.tsx

```diff
@@ -22,9 +22,11 @@ import { EditComponentProps } from '../BaseCredentialsSection';
 export const EsxiCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onChange }) => {
   const { t } = useForkliftTranslation();
 
-  const user = safeBase64Decode(secret?.data?.user || '');
+  const user = secret?.data?.user ? safeBase64Decode(secret?.data?.user) : secret?.data?.user;
```

Comment:
`safeBase64Decode` is here so you will not need to do` x ? Base64.decode(x) : undefined`
can you find a way to keep the simplicity of: `x = safeBase64Decode(y)` ?